Dockerfile

```diff
@@ -1,16 +1,11 @@
-FROM node:18-alpine
+FROM node:20.4.0-alpine3.18
 
-RUN apk add --no-cache python3 make gcc g++ 
 WORKDIR /app
 
-COPY "docker/entrypoint.sh" ./
-
 COPY package*.json ./
-RUN npm install bcrypt && npm rebuild bcrypt --build-from-source
 RUN npm install
 
 COPY . ./

```

Comment:
This will copy the `node_modules` folder too if it exists will it not? Doing so will potentially overwrite the modules installed above

Reply:
`node_modules` is in the `.dockerignore` file, so it should be excluded.